server/src/index.ts

```diff
@@ -7,7 +7,8 @@ const sql = require('./config/databaseHandler');
 const app = express();
 const PORT = 8000;
 
-let userRoute = require('./routes/users-routes')
+let userRoute = require('./routes/user-routes')
+let paymentRoute = require('./routes/payment-routes')
 
```

Comment:
probably should add group-expense and group routes here

Reply:
I agree with you @Abeilles14, I assumed that whoever was working on those would add them in when they finished those routes